modules/mono/csharp_script.cpp

```diff
@@ -30,7 +30,18 @@
 
 #include "csharp_script.h"
 
-#include <stdint.h>
+#include "godotsharp_dirs.h"
+#include "managed_callable.h"
+#include "mono_gd/gd_mono_cache.h"
+#include "signal_awaiter_utils.h"
+#include "utils/macros.h"
+#include "utils/naming_utils.h"
+#include "utils/path_utils.h"
+#include "utils/string_utils.h"
+
+#ifdef DEBUG_METHODS_ENABLED
+#include "class_db_api_json.h"
+#endif
```

Comment:
In the end, position of all these conditional sections is pretty arbitrary and not consistent. It feels right being here and not below, but it doesn't follow any rule and other files are done differently. Hmm.

Reply:
There are more below too, for the record.

Reply:
I guess you've answered above :)

Reply:
It's the same as https://github.com/godotengine/godot/pull/75944/files/db2ec81de925e8dd6e90d62901f19e8161f57adc#r1230850389

It's all consistent in that framework I think.